index.html

```diff
@@ -0,0 +1,288 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Checkout Page</title>
+	<meta charset="utf-8">
+	<link rel="stylesheet" type="text/css" href="css/main.css">
+	<link rel="stylesheet" type="text/css" href="css/fonts/fonts.css">
+</head>
+<body>
+<header>
+	<div class="col-3">
+		<button class="main_btn goBack">Go back</button>
+	</div>
+	<div class="col-3">
+		<p class="logo"><img src="css/img/kottans.png"></p>
+	</div>
+	<div class="col-3 sign">
+		<button class="main_btn btn_sgn">Sign In</button>
+	</div>
+</header>
+
+
+<section class="yourOrder">
+	<div class="col-6">
+		<div class="details">
+			<div class="blank"><!--<p><img src="https://png.icons8.com/material/26/6c748d/file.png"></p>--> <p>Your order</p></div>
+			<div>
+				<ul class="list">
+					<li><span>Items</span></br></br>2 documents</li>
+					<li><span>Details</span></br></br>2 languages</li>
+					<li><span>Price</span></br></br>$ 696.06</li>
+					<li><span>Delivery</span></br></br>48 hours*</li>
+				</ul>
```

Comment:
this looks like a perfect case to use table element (list usually contains similar elements)